src/App.svelte

```diff
@@ -49,12 +50,12 @@
     showAbout = false;
   }
 
-  let boundaryGeojson;
+  let boundaryGeojson: FeatureCollection<Polygon>;
   onMount(async () => {
     boundaryGeojson = await loadAuthorityBoundary();
   });
 
-  async function loadAuthorityBoundary() {
+  async function loadAuthorityBoundary(): Promise<FeatureCollection<Polygon>> {
```

Comment:
Things like this are a little curious to me. Nothing checks the return type is correct; TS appears to just trust us